### ISSUE TITLE
Spirit World (Avatar TLA)

### ISSUE THREAD

**Issue by Phantom329** [Azanor/thaumcraft-suggestions#113]
I had an idea for an alternate overworld dimension with increased magical properties, grow ticks, etc. The idea comes from 'Avatar the last Airbender' series and you would be able to get there initially in spirit, similar to dreams from witchery (haven't played that mod in a while though).

You would not keep your inventory and it would be useful for research mostly. Don't spawn endermen so that you can't craft enderchests. Maybe have higher chances of certain events or whatever. Have very low ore gen, maybe only vanilla ores plus TC ores; just enough to get a basic research area.

As a mid-late game have a craftable portal to 'properly' get to the dimension and keep inventory on travel.

Think of how much creative potential there is in a whole other dimension based on current lore!
A few other ideas:
-Center area is balanced while some radius outward is tainted/fluxed/sinister/other
-Initially only visit in adventure mode (to do research)
-1/4 scale like the Nether for fast travel
-Unique fauna/flora for cool items; miniboss level elementals, hostile when attacked, turn the environment against you
-Bad things happen if you damage living things too much, flux/taint/fauna become hostile
-Everything changed when the Cultist nation attacked...

**Comment by cadaverous-eris** [comment]
...sooooo the spirit world from witchery?

**Comment by Phantom329** [comment]
I never actually used witchery I just know it exists.

**Comment by cadaverous-eris** [comment]
Fair enough. Worth checking out if you want to play around with a spirit dimension

**Comment by Phantom329** [comment]
I actually haven't played in a long time, mostly waiting for TC 6 release. I still think it would be an awesome dimension, Azanor has a certain quality in his mods that I have not seen in many other mod.

Also, witchery is meh anyway.

**Comment by cadaverous-eris** [comment]
What? How can you tell if you've never used it?! Whatever, though. You do you.

**Comment by Phantom329** [comment]
I mean that I've used it enough to know its meh which is why I don't use it.

**Comment by cadaverous-eris** [comment]
Fair enough
